README.md

```diff
@@ -1,43 +1 @@
 # Vac.dev Website
-
```

Comment:
Can we add instructions for how to run this locally?

Reply:
Sure, I will do this after the first launch :) 
Although for that you would need to clone and run a different repository! This repo is purely for content purposes.